.github/workflows/scheduled_builds.yml

```diff
@@ -1,60 +1,16 @@
-name: "Build Edge on Schedule"
+name: Build Edge on Schedule
 
 on:
   schedule:
-    - cron: "0 0 * * 5"
+    - cron: 0 0 * * 5
 
 permissions:
   contents: read
+  packages: write
```

Comment:
Why was it necessary to add this permission? What changed for this workflow?

Reply:
I have never understood why we can push an image to GHCR in the `scheduled_builds` workflow without this permission. Is `packages` the wrong keyword here?

Reply:
Aha, please have a look at: https://github.com/docker-mailserver/docker-mailserver/actions/workflows/scheduled_builds.yml

It turns out, without someone noticing it, that we actually need this :D